reV/supply_curve/supply_curve.py

```diff
@@ -70,7 +74,10 @@ def __init__(self, sc_points, trans_table, fcr, sc_features=None,
         self._consider_friction = consider_friction
         self._calculate_total_lcoe_friction()
 
+        mask = ~pd.isna(self._trans_table['sc_gid'])
```

Comment:
Turn this into a static method that would return trans_table, sc_gids, and mask

Reply:
Agreed!